include/sparrow/utils/vector_view.hpp

```diff
@@ -30,12 +30,12 @@ namespace sparrow
      * to string_view, but for arbitrary T. You can consider it as a span or range supporting
      * const operations only, and comparison operators.
      */
-    template <class T>
-    class vector_view : public std::span<T>
+    template <class T, std::size_t Extent = std::dynamic_extent>
+    class vector_view : public std::span<T, Extent>
```

Comment:
Why this change? I don't see it used anywhere (not that I'm agains it, but if not related to the fixed_width_binary, it should be in a dedicated PR).

Reply:
This modification was mandatory at some point but I changed my modification later. I will remove this modification.

Reply:
Fixed